components/src/core/list-item-tag/list-item-tag.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ListItemTagComponent } from './list-item-tag.component';
+import { ListItemTagModule } from './list-item-tag.module';
+
+describe('ListItemTagComponent', () => {
+  let component: ListItemTagComponent;
+  let fixture: ComponentFixture<ListItemTagComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [ListItemTagModule],
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ListItemTagComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
We need a test to enforce class names, see the other components for reference here.

Reply:
Fixed with commit 8327066.